ResumeAR/Credentials.swift

```diff
@@ -2,20 +2,22 @@
 //  Credentials.swift
 //  ResumeAR
 //
-//  Created by Sanjeev Ghimire on 11/1/17.
-//  Copyright © 2017 Sanjeev Ghimire. All rights reserved.
+//  Created by Sanjeev Ghimire on 4/9/18.
+//  Copyright © 2018 Sanjeev Ghimire. All rights reserved.
```

Comment:
Generally don't move copyrights forward.  The initial copyright would be the one that matters. Also we don't do individual copyrights at IBM. We use the IBM template.

Reply:
I removed it and created a new one again. I created  `CredentialsSample.swift` but then the code wouldn't compile it the first place as it references to `Credentials` struct. So hence the new copyright.

Reply:
I am researching on how  I can create an environment variables in swift which can be read and `Credentials.swift` would be populated. That might be an enhancement to this.

Reply:
an ibm copyright would look like this example from javascript
/**
 * Copyright 2017 IBM Corp. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
 * use this file except in compliance with the License. You may obtain a copy of
 * the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 * License for the specific language governing permissions and limitations under
 * the License.
 */

Reply:
so all the class has to have this instead of personal copyright?

Reply:
IBM owns our original work. If the original source was somewhere else, then we'd want to bring that copyright forward (and follow the license requirements for derived works, modifications, etc...).

Reply:
okay I will add the copy right to all the classes

Reply:
done.